cmd/internal/cli/apptainer.go

```diff
@@ -63,10 +63,10 @@ var (
 )
 
 const (
-	envPrefix = "SINGULARITY_"
+	envPrefix = "APPTAINER_"
```

Comment:
No, we need to retain backward compatibility for environment variables.

Reply:
I invite us to defer this until resolution of issue #7 and make #7 a blocker for v1.0.0

Reply:
Yes so in the meanwhile I think no change should be made to environment variables.

Reply:
deferred to issue #7